### PR TITLE
docs:  updated edge docs PE-2801

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR removes the PE argument from the reference documentation.

## Review Changes

💻 [Preview URL](https://deploy-preview-1655--docs-spectrocloud.netlify.app/clusters/edge/edgeforge-workflow/palette-canvos)

🎫 [PE-2801](https://spectrocloud.atlassian.net/browse/PE-2801)

### CODE REVIEW

File: docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos.md [hunk 1]
```diff
@@ -721,41 +738,51 @@ Use the following instructions on your Linux machine to customize the arguments
   |OS|Public Repo|BYOS Edge OS|`1.0.0`|
 
 
-21. Replace the the cluster profile's BYOOS pack manifest with the following custom manifest so that the cluster profile can pull the provider image from the ttl.sh image registry.  
+21. Replace the the cluster profile's BYOOS pack manifest with the output provided to your earlier that you copied.
```
Comment (ritawatson) [inline docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos.md:741]: Line 724 reads awkwardly or something is missing. "your" should probably be "you". But the larger question is whether the user is provided something or has copied something. This needs clarifying.

Comment (ritawatson) [inline docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos.md:741]:
```suggestion
21. Replace the cluster profile's BYOOS pack manifest with the output provided to your earlier that you copied.  
```

**Review by ritawatson — APPROVED** [review]
There's a sentence in the palette-canvos.md doc that needs attention. Otherwise, LGTM.

File: docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos.md [hunk 2]
```diff
@@ -721,41 +738,51 @@ Use the following instructions on your Linux machine to customize the arguments
   |OS|Public Repo|BYOS Edge OS|`1.0.0`|
 
 
-21. Replace the the cluster profile's BYOOS pack manifest with the following custom manifest so that the cluster profile can pull the provider image from the ttl.sh image registry.  
+21. Replace the cluster profile's BYOOS pack manifest with the output provided to your earlier that you copied.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos.md:741]:
```suggestion
21. Replace the cluster profile's BYOOS pack manifest with the output that was provided to you earlier and that you copied.
```

**Pull request merged**